src/test/java/core/basesyntax/service/impl/ReportCreatorServiceImplTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Storage;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportCreatorServiceImplTest {
+    private static ReportCreatorServiceImpl reportCreatorService;
+    private static String expected1;
+    private static String expected2;
+
+    @BeforeClass
+    public static void beforeClass() {
+        reportCreatorService = new ReportCreatorServiceImpl();
+        expected1 = "fruit,quantity" + System.lineSeparator()
+                + "banana,152" + System.lineSeparator()
+                + "apple,90";
+        expected2 = "fruit,quantity";
+    }
+
+    @Test
+    public void createReport_isValid_Ok() {
+        Storage.mapFruits.put("banana", 152);
+        Storage.mapFruits.put("apple", 90);
+        String actual = reportCreatorService.createReport();
+        boolean isEqual = actual.equals(expected1);
+        assertTrue(isEqual);

```

Comment:
maybe better to invoke methods in fruit shop to generate map and create report instead of hard code result 
it doesn`t need to change it is only my opinion

Reply:
i simply add some pairs to empty map to get expected string.. for me it's not a problem)